rnote-ui/data/ui/filerow.ui

```diff
@@ -66,8 +66,16 @@
             <attribute name="label" translatable="yes">Move into trash</attribute>
             <attribute name="action">filerow.trash-file</attribute>
           </item>
+          <item>
+            <attribute name="label" translatable="yes">Duplicate</attribute>
+            <attribute name="action">filerow.duplicate</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="yes">Create Directory</attribute>
+            <attribute name="action">filerow.create-dir</attribute>
+          </item>
```

Comment:
The create directory action should probably be a button or in a context menu when clicking between the files, not an entry for every file. I can change that afterwards, since you said you are not too familiar with GTK and you probably want to have this PR merged.

Reply:
> The create directory action should probably be a button or in a context menu when clicking between the files, not an entry for every file.

Ah, I understand what you mean.

>  I can change that afterwards, since you said you are not too familiar with GTK and you probably want to have this PR merged.

If you want, we can create a  new issue to keep track of this. I'm willing to learn a bit more about GTK, which tutorial would you recommend to get started with it? I can try to create another PR with "fixes" this. :)

Reply:
Sure, sounds good to me. :) I listed some resources that helped me learning GTK here: https://github.com/flxzt/rnote/discussions/108 . Especially the rust gtk book is a nice introduction

Reply:
So should I fix that in a following PR, so we can merge this one first or should I add those changes to this one?

Reply:
@flxzt (ping)

Reply:
I think having it in this one would be best!

Reply:
> The create directory action should probably be a button or in a context menu when clicking between the files, not an entry for every file.

May I ask, where you want to see it?

In my opinion, or at least what I find intuitive is to place this "Add directory" button here:
![image](https://user-images.githubusercontent.com/50843046/195228408-8a288493-f374-44f8-83ae-13e8b203a856.png)

but the "problem" is, that there's already other stuff like donating and so on. Where should I put the button?

Reply:
@flxzt (ping)

Reply:
I was thinking adding a button for a "directory modification" dropdown menu here:
![Bildschirmfoto vom 2022-10-12 07-05-56](https://user-images.githubusercontent.com/19841886/195254896-6b88ac09-da30-4ade-b458-c04eb21ddf1d.png)

I would wrap [this widget](https://github.com/flxzt/rnote/blob/main/rnote-ui/data/ui/workspacebrowser.ui#L33) in a horizontal box and add a sibling "MenuButton" to it. Check how the menubutton + dropdown menu is [implemented for the filerow](https://github.com/flxzt/rnote/blob/main/rnote-ui/data/ui/filerow.ui#L33), you can copy and paste it from there for the most part.
(And don't worry about the icon, you can just reuse the vertical `...`, this can be tweaked later)